packages/renderer/src/stores/navigation/navigation-registry.ts

```
@@ -75,24 +73,31 @@ const init = (): void => {
 
-function collecItem(navigationRegistryEntry: NavigationRegistryEntry, items: DisplayItem[]): void {
+export function collecItem(
+  navigationRegistryEntry: NavigationRegistryEntry,
+  items: NavigationItemState[],
+  groupName?: string,
+): void {
   if (navigationRegistryEntry.items && navigationRegistryEntry.type === 'group') {
-    navigationRegistryEntry.items.forEach(item => {
-      collecItem(item, items);
-    });
+    navigationRegistryEntry.items.forEach(item => collecItem(item, items, groupName));
```

Biome (2.5.3)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)